CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+# Contributing Guidelines
+
+There are many ways to contribute to CoW Protocol. We welcome contributions of all kinds from anyone, whether you've been a long-time user or you are just getting started with development.
+
+## Troubleshooting
+
+You can help other users in the community to solve their issues in the [CoW Protocol Discord].
+
+[CoW Protocol Discord]: https://discord.gg/cowprotocol
+[CoW Protocol Forums]: https://forum.cow.fi
+
+## Opening an issue
+
+You can [open an issue] to suggest a feature or report a minor bug.
+
+If you believe your issue may be due to user error and not a problem in the library, consider instead posting a question on the [CoW Protocol Discord].
+
+Before opening an issue, be sure to search through the existing open and closed issues, and consider posting a comment in one of those instead.
+
+When requesting a new feature, include as many details as you can, especially around the use cases that motivate it. 
+Features are prioritised according to the impact they may have on the ecosystem, so we appreciate information showing that the impact could be high.
+
+[open an issue]: ./issues/new/choose
+
+## Submitting a pull request
+
+If you would like to contribute code or documentation you may do so by forking the repository and submitting a pull request.
+
+Any non-trivial documentation must be first discussed with the maintainers in an issue (see [Opening an issue](#opening-an-issue)). Only very minor changes are accepted without prior discussion.
+
+When opening the pull request you will be presented with a template and a series of instructions. Read through it carefully and follow all the steps. 
+Expect a review and feedback from the maintainers afterwards.
+
+If you're looking for a good place to start, look for issues labelled ["help wanted"](./labels/help%20wanted)!
+
+## Reward for Contributions
+We are excited to support our community's contributions in a tangible way. 
+For merged pull requests that contribute to issues labeled "help wanted", we are offering a reward of 100 DAI.
+To be eligible for this reward, make sure your pull request meets all the required criteria and is successfully merged. 
```

Comment:
If I were a contributor reading this I would like to have a little bit more information on the process.
Primarily on which chain and by which way we reach out the user.